### PR TITLE
Give Logstash more time to shut down

### PR DESCRIPTION
**Pull request by ycombinator**
ESTF Logstash Monitoring Parity Test build jobs are [failing](https://internal-ci.elastic.co/job/elastic+estf-monitoring-snapshots+7.x+multijob-logstash/4/console) with the following error:

```
03:04:38 TASK [logstash : Fail if logstash process is running] **************************
03:04:38 fatal: [aithost]: FAILED! => {
03:04:38     "changed": false
03:04:38 }
03:04:38 
03:04:38 MSG:
03:04:38 
03:04:38 logstash is running
```

I was able to reproduce this failure locally but not all the time. So this is a flaky test failure, similar to the one we saw for Elasticsearch in #263.

Applying the same solution as in #263, this PR gives Logstash 15 seconds more to shut down, by re-checking for it's PID every 3 seconds up to 5 times.

### CODE REVIEW

File: ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@
       args:
         executable: /bin/bash
       register: logstash_process_id
+      retries: 5
+      delay: 3
+      until: elasticsearch_process_id.stdout == ""
```
Comment (liza-mae) [inline ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: This should be logstash_process_id.stdout not elasticsearch_process_id.stdout.  
You can remove the following step 'Fail if logstash process is running' and take it out of the block statement.

Comment (ycombinator) [reply ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: Good eye, @liza-mae! Thanks!

Comment (ycombinator) [reply ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: I've fixed the typo in bf95a7d.

@liza-mae could you explain why the 'Fail if logstash process is running` step is not necessary? BTW, there's a similar step for Elasticsearch here: https://github.com/elastic/elastic-stack-testing/blob/master/ansible/roles/elasticsearch/tasks/linux/elasticsearch_is_not_running.yml#L18-L21.

Comment (liza-mae) [reply ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: oh yea it is a redundant check, if the process is still running the retry until condition will fail so the next statement won't be executed anyway.

Comment (liza-mae) [reply ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: We can do another PR to clean it up, if you prefer.

Comment (ycombinator) [reply ansible/roles/logstash/tasks/linux/logstash_is_not_running.yml]: Thanks, I've removed the redundant check for LS and ES in 1031dee.

**Pull request merged**